packages/design-system/src/components/TextField/Mask.test.interaction.ts

```diff
@@ -0,0 +1,26 @@
+import { test, expect } from '@playwright/test';
+import themes from '../../../../../themes.json';
+
+const sbID = 'components-text-field--label-masked-ssn';
+const storyUrl = `http://localhost:6006/iframe.html?viewMode=story&id=${sbID}`;
+
+Object.keys(themes).forEach((theme) => {
+  if (themes[theme].incomplete) return;
+
+  test(`Mask partial text entry: ${theme}`, async ({ page }) => {
```

Comment:
I was going to suggest this title be specific to SSNs, but I guess the SSNs are a superset of all mask functionality